src/app/styles/globals.css

```diff
@@ -85,3 +85,12 @@ input[type="number"]::-ms-clear {
 [data-theme] {
   background-color: hsl(var(--b2) / var(--tw-bg-opacity, 1));
 }
+
+/* Grid layout for home page */
+.custom-auto-row-grid[data-path="/"] {
```

Comment:
the path for this page will later change to `/trade` (and the landing page `/` will have different content, if the custom class names are unavoidable, please add another name and use it for the trading layout somewhere. you can add a div to wrap the trade page content if it doesn't exist yet

Reply:
it's alright, when it will be moved to the trade page, we just need to change this :

`.custom-auto-row-grid[data-path="/"]`

to this: 

`.custom-auto-row-grid[data-path="/trade"]`

and the layout will work fine.

Reply:
ok